<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreMenu</class>
 <widget class="QDialog" name="ScoreMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_In">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Exe">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pic/seu.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_Modify">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>修改成绩</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_Search">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>查询成绩</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
